### DISCUSSION TITLE
please i new to all this things coding

### DISCUSSION THREAD

**Discussion by omohro12** [WalletConnect/walletconnect-monorepo#2659] [Developer Support] [answered]
Good morning, I need for person to set this up for me v-1 to v-2 walletconnect migration, I don't know how to do it, I hope my fund wouldn't get lost? admin should help

**Comment by glitch-txs** [comment]
Hi, are you building a dapp or a wallet? If you are a wallet user is not up to you to update but the wallet your using

**Comment by omohro12** [comment]
but if I don't upgrade would it not affect my account?

**Reply by glitch-txs** [reply]
What wallet are you using? You can always move your private key to another wallet if there's any issue, but I doubt you'll need to

**Comment by omohro12** [comment]
rainbow wallet

**Reply by glitch-txs** [reply]
Cool, Rainbow already supports WalletConnect v2, so don't worry about it, just make sure the app version is up to date :)

**Comment by omohro12** [comment]
how can you move your private key to another wallet apart from the wallet
that owns that key?

**Comment by omohro12** [comment]
you mean I just not need anything to do anything for now, is just for me to
wait and see when the migration will happen, that I don't need to migrate
my account, is that what you mean?

**Reply by glitch-txs** [reply, marked as answer]
You don't need to migrate because the wallet already did. Rainbow already supports v2

Yes, you don't need to do anything 🙂
